sdk/identity/Azure.Identity.Broker/api/Azure.Identity.Broker.net462.cs

```diff
@@ -9,6 +9,6 @@ public partial class SharedTokenCacheCredentialBrokerOptions : Azure.Identity.Sh
     {
         public SharedTokenCacheCredentialBrokerOptions() { }
         public SharedTokenCacheCredentialBrokerOptions(Azure.Identity.TokenCachePersistenceOptions tokenCacheOptions) { }
-        public bool? IsMsaPassthroughEnabled { get { throw null; } set { } }
+        public bool? IsLegacyMsaPassthroughEnabled { get { throw null; } set { } }
```

Comment:
Curious how this is related to this change?

Reply:
@annelo-msft no idea, but running `eng\scripts\Export-API.ps1` updated this.